### PR TITLE
Release k6chaijs@4.5.0.1

### PR DESCRIPTION
**Pull request by Blinkuu**
## Description

Release k6chaijs@4.5.0.1. This version adds support for `exitOnError" option. For more details, please see https://github.com/grafana/k6-jslib-k6chaijs/commit/8eddf2dc6369fdecb9fe2a4854d46afe833afb31.

- [x] Use a meaningful title for the Pull Request. Include the name of the jslib added/modified.
- [x] Fill the description section of the Pull Request. 
- [x] Test the change in your code, and ensure the `npm run test` command succeeds.
- [x] Run `yarn run generate-homepage` locally and verify the new homepage `/lib/index.html` file looks legit.

*If publishing a new version of an existing jslib*:
- [x] The Pull Request is labeled with the `version bump` label.
- [x] The Pull Request adds a `/lib/{jslib_name}/{desired_version}` folder.
- [x] The Pull Request adds a `/lib/{jslib_name}/{desired_version}/index.js` file containing the jslib's code bundle.
- [x] The Pull Request updates the `supported.json` file to contain an entry for the newly added jslib version, as in the following example:
```JSON
{
  "my-lib": [
    "1.0.2",
    // Use semantic versioning
    "{desired_version}"
  ]
}
```
- [x] The Pull Request adds the relevant tests to the `/tests/basic.js` and `/tests/testSuite.js` files to ensure that the new version of the jslib is importable and runnable by k6.

### CODE REVIEW

**Review by joanlopez — COMMENTED** [review]
I'm just wondering, if we have added a new configuration/behavior, shouldn't it been v4.5.1, or even v4.6.0? 🤔

File: tests/k6chai.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { expect as expect4340, describe as describe4340 } from '../lib/k6chaijs/
 import { expect as expect4341, describe as describe4341 } from '../lib/k6chaijs/4.3.4.1/index.js'
 import { expect as expect4342, describe as describe4342 } from '../lib/k6chaijs/4.3.4.2/index.js'
 import { expect as expect4343, describe as describe4343 } from '../lib/k6chaijs/4.3.4.3/index.js'
+import { expect as expect4501, describe as describe4501 } from '../lib/k6chaijs/4.5.0.1/index.js'
```
Comment (joanlopez) [inline tests/k6chai.js:5]: I see these changes are missing for `v4.5.0.0`, so perhaps this is a good opportunity to add them? 🤔

**Comment by Blinkuu** [comment]
> I'm just wondering, if we have added a new configuration/behavior, shouldn't it been v4.5.1, or even v4.6.0? 🤔 Or this doesn't follow some sort of semantic versioning?

@joanlopez In the README, we can see that our version should follow the chai version (see [here](https://github.com/grafana/k6-jslib-k6chaijs?tab=readme-ov-file#deploy-new-version)). Since chai versions use only MAJOR, MINOR and PATCH numbers (in other words, the semver is in `X.Y.Z` format) I figured that bumping the fourth number makes sense to indicate it.

File: tests/k6chai.js (hunk 1 above)
Comment (Blinkuu) [reply tests/k6chai.js:6]: Fixed in [32c36c2](https://github.com/grafana/jslib.k6.io/pull/139/commits/32c36c2f5a91bb10817d9d77eba490d62ea79335).

Comment (joanlopez) [reply tests/k6chai.js:6]: Great, thanks! :)

**Comment by joanlopez** [comment]
> > I'm just wondering, if we have added a new configuration/behavior, shouldn't it been v4.5.1, or even v4.6.0? 🤔 Or this doesn't follow some sort of semantic versioning?
> 
> @joanlopez In the README, we can see that our version should follow the chai version (see [here](https://github.com/grafana/k6-jslib-k6chaijs?tab=readme-ov-file#deploy-new-version)). Since chai versions use only MAJOR, MINOR and PATCH numbers (in other words, the semver is in `X.Y.Z` format) I figured that bumping the fourth number makes sense to indicate the change in our functionality.

Ah, ah! Right! My mistake, and now that you mentioned, I remember to have the same thought and reading the same comment, so double fault! Apologies! 🙏🏻

**Review by joanlopez — APPROVED** [review]

**Pull request merged**